Portal/src/Datahub.Portal/Pages/Workspace/Publishing/PublishingDashboard.razor

```diff
@@ -0,0 +1,165 @@
+﻿@* @page "/w/{ProjectAcronym}/publishing" *@
+@using Datahub.Application.Services.Publishing;
+@using Datahub.Application.Services.Security;
+@using Datahub.Core.Model.Projects;
+@inject IDbContextFactory<DatahubProjectDBContext> _dbContextFactory
+@inject IOpenDataPublishingService _publishingService
+@inject ITbsOpenDataService _tbsService;
+@inject IDialogService _dialogService;
+@inject NavigationManager _navManager;
+@inject IKeyVaultUserService _kvService;
+@inject Microsoft.Identity.Web.MicrosoftIdentityConsentAndConditionalAccessHandler _consentHandler
+
+
+@if(IsPublishingEnabled)
+{
+    <MudText Typo=@Typo.h2>@Localizer["Publishing Dashboard"]</MudText>
+
+    <MudStack Spacing="6">
+
+        <MudText>
+            <MudButton OnClick=@PublishNewDataset Variant=@Variant.Filled Color=@Color.Primary>@Localizer["Publish New Dataset"]</MudButton>
+        </MudText>
```

Comment:
Without it, the button expands to the full page width. There may be a more semantically correct component to wrap it with to avoid that, but MudText makes it normal sized.